File: SofiForce.Models/Models/EntityModels/VendorModel.cs
using System;

namespace Models
{
    public class VendorModel
    {
        public int VendorId { get; set; }
        public string VendorCode { get; set; }
        public int? VendorGroupId { get; set; }
        public string VendorNameEn { get; set; }
        public string VendorNameAr { get; set; }
        public bool? IsLocal { get; set; }
        public bool? IsActive { get; set; }
        public int? DisplayOrder { get; set; }
        public string Color { get; set; }
        public string Icon { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/VendorModel.ts
import { VendorGroupModel } from "./vendorGroupModel";
import { ItemModel } from "./itemModel";

export interface VendorModel {
    vendorId: number;
    vendorCode: string;
    vendorGroupId: number | null;
    vendorNameEn: string;
    vendorNameAr: string;
    isLocal: boolean | null;
    isActive: boolean | null;
    displayOrder: number | null;
    color: string;
    icon: string;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}